services/ui-src/src/verbiage/errors.ts

```diff
@@ -14,6 +14,8 @@ export const validationErrors = {
   INVALID_URL: "Response must be a valid hyperlink/URL",
   INVALID_DATE: "Response must be a valid date",
   INVALID_END_DATE: "End date can't be before start date",
+  NUMBER_LESS_THAN_ONE: "Response must be equal to or greater than one",
+  NUMBER_LESS_THAN_ZERO: "Response must be equal to or greater than zero",
```

Comment:
Should this be 
```suggestion
  NUMBER_GREATER_THAN_ONE: "Response must be equal to or greater than one",
  NUMBER_GREATER_THAN_ZERO: "Response must be equal to or greater than zero",
```

Reply:
Or NUMBER_NOT_LESS_THAN_ONE to follow the above naming scheme

Reply:
@ntsummers1 This brings up a good question, I wanted to write it as an error so when its used in situations like `message: error.NUMBER_LESS_THAN_ONE,` we know whats wrong with the input.